index.js

```diff
@@ -16,18 +16,32 @@ jszip.external.Promise = bluebird;
 
 const DEFAULT_EXCLUDE = ['node_modules/serverless-cljs-plugin/**'];
 
-function destroyFnMap(service) {
+// We have to exclude the lumo binary explicitely:
+// https://github.com/serverless/serverless/issues/4359
+const LUMO_EXCLUDE = ['.lumo_cache/**', 'node_modules/.bin/lumo'];
+
+function setCljsLambdaFnMap(serverless) {
   return _.mapValues(
-    service.functions,
+    serverless.service.functions,
     fn => {
       if(fn.cljs) {
         fn.handler = `index.${munge.munge(fn.cljs)}`;
-        _.set(fn, 'package.artifact', service.__cljsArtifact);
+        if(!isLumo(serverless)) {
+          _.set(fn, 'package.artifact', serverless.service.__cljsArtifact);
+        }
       }
       return fn;
     });
 }
 
+function setCljsLambdaExclude(serverless) {
+  let newExclude = DEFAULT_EXCLUDE;
+  if (isLumo(serverless)) {
+    newExclude = _.concat(newExclude, LUMO_EXCLUDE);
+  }
+  return _.update(serverless.service, 'package.exclude', v => _.concat(v || [], newExclude));
```

Comment:
I don't think it's a big deal if we just put the `LUMO_EXCLUDE` entries in `DEFAULT_EXCLUDE`, and remove the conditional in this function.